<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Height and Weight Converter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="kilo_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kilograms:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pounds:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pound_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ounces:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ounce_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="meter_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="feet_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="centimeter_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	border: 1px solid black;
	border-radius: 15px;
	padding: 5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Feet:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Centimeters:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Meters:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
